pkg/network/state.go

```diff
@@ -208,8 +208,7 @@ func (ns *networkState) GetDelta(
 	}
 	ns.clients[id].stats = newStats
 
-	// Flush closed connection map and stats
-	ns.clients[id].closedConnections = map[string]ConnectionStats{}
+	ns.clients[id].closedConnections = nil
```

Comment:
If we assume `closedConnections` is initialized, isn't this a bad idea to set to `nil`?